Feature/37159 temp upload mime type whitelist (#397)
valtimo-platform/valtimo-documentation: release-notes/major10/10.7.0/valtimo-backend-libraries.md

  [TaakStatus](https://github.com/valtimo-platform/valtimo-backend-libraries/blob/main/zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/TaakObject.kt#L49)
  enum of a TaakObject.

> * **Whitelisting by mime-type when for uploads to temporary file storage**

Reviewer: ```suggestion
* **Whitelisting by mime-type of uploads to temporary file storage**
```

>   Uploads to the temporary file storage can now be restricted by whitelisting allowed mime-types.

Reviewer: ```suggestion
The types of files uploaded to the temporary file storage can now be restricted by whitelisting allowed mime-types.
```

>   For more information on this feature can be found [here](/using-valtimo/upload/temporary-file-storage.md#whitelisting-file-types-for-uploads)

Reviewer: ```suggestion
More information on this feature can be found [here](/using-valtimo/upload/temporary-file-storage.md#whitelisting-file-types-for-uploads)
```
